docker/recorder/src/main/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironment.java

```diff
@@ -186,6 +201,59 @@ private GroupEntry createDockerCompositionSchema(CubeDockerConfiguration cubeDoc
         return containersComposition;
     }
 
+    private GroupEntry createNetworkTopologyGraph(CubeDockerConfiguration cubeDockerConfiguration, ReporterConfiguration reporterConfiguration, DockerClientExecutor executor) {
+
+        final GroupEntry netTopology = new GroupEntry("Network Topology Graph");
+        mxGraph graph = new mxGraph();
+        Object parent = graph.getDefaultParent();
+
+        graph.setAutoSizeCells(true);
+        graph.getModel().beginUpdate();
+        try {
+            DockerCompositions dockerCompositions = cubeDockerConfiguration.getDockerContainersContent();
+            final Map<String, CubeContainer> containers = dockerCompositions.getContainers();
+            final Map<String, Object> insertedVertex = new HashMap<>();
+                for (Map.Entry<String, CubeContainer> container: containers.entrySet()) {
+                    final String containerId = container.getKey();
+                    Object containerName = graph.insertVertex(parent, null , containerId, 0, 0, 80, 30);
+                    final CubeContainer cubeContainer = container.getValue();
+                    Set<String> nwList = new HashSet<>();
+                    if (cubeContainer.getNetworkMode() != null) {
+                        nwList.add(cubeContainer.getNetworkMode());
+                    } else {
+                        InspectContainerResponse inspect = executor.inspectContainer(containerId);
```

Comment:
If you look at [this](https://github.com/dipak-pawar/arquillian-cube/blob/39bdd2d60bdffc00e083f71f55ab8b6e0c394f4a/docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/compose/ComposeBuilder.java#L50) we are assigning value of networkmode if there is no networks in docker-compose.yml.

but in case of CUBE format we are not setting it to cubecontainer, so networkmode is null for cubecontainer. but as there is no networkmode mentioned in containerdefinition, docker-host is assigning it as default. So we are getting it null as we are not setting it, docker host is setting it by default.
So to get default networkmode, we need to query this.